Bottom: fe0cdecc214254185fc2b69fde41844205035588
Top:    1a20a1fb4126a24bc74afc0d42a8967e461c4afd
Author: Parth Shah <parth@linux.ibm.com>
Date:   2019-07-09 12:00:15 +0530

Refresh of sched-fair-remove-sis_fold

---

diff --git a/kernel/sched/features.h b/kernel/sched/features.h
index a451b3d4ecd1..d291bba357ba 100644
--- a/kernel/sched/features.h
+++ b/kernel/sched/features.h
@@ -53,10 +53,10 @@ SCHED_FEAT(NONTASK_CAPACITY, true)
 SCHED_FEAT(TTWU_QUEUE, true)
 
 /*
- * When doing wakeups, attempt to limit superfluous scans of the LLC domain.
+ * When doing wakeups, attempt to limit scanning cost of the LLC proportional to
+ * the average idle time.
  */
 SCHED_FEAT(SIS_PROP, true)
-SCHED_FEAT(SIS_FOLD, true)
 
 /*
  * Issue a WARN when we do multiple update_rq_clock() calls
